sources/packages/web/src/services/http/UserApi.ts

```diff
@@ -9,13 +9,26 @@ export default interface User {
 }
 
 export class UserApi extends HttpBaseClient {
-  public async checkUser(): Promise<User> {
+  public async checkUser(): Promise<boolean> {
+   
     try {
-      const response = await this.apiClient.get(
+      return await this.apiClient.get(
         "users/check-user",
         this.addAuthHeader(),
       );
-      return response.data as User;
+      
+    } catch (error) {
+      this.handleRequestError(error);
+      throw error;
+    }
+  }
+
+  public async synchronizeUserInfo(): Promise<void> {
+    try {
+      /**
+       * TODO Dont quite agree with sending a null in a patch/put request but going with it for now
```

Comment:
Please propose an alternative method that would make more sense or please consider remove this comment.